airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/test/models.py

```diff
@@ -1,11 +1,14 @@
 # Copyright (c) 2023 Airbyte, Inc., all rights reserved.
 
 import os
-from typing import Dict, Set
+from typing import Dict, List, Set
 
 from pydantic import BaseModel, Field, validator
 
 
+DEFAULT_PYTHON_VERSION = "3.10.12"
```

Comment:
Ironically, while the default here is / was 3.10, connectors run 3.9 today. Nothing for you to change, just our mistake.

Reply:
They still do. Connector testing is a bit on the side compared to so-called component testing. That's different commands on airbyte-ci and they bootstrap dagger differently.. From what I figured, it's probably cause you have prebuilt Docker to inherit for components that they run into. I wonder if it's only used for testing or if it's a part of distribution and deployment. If it's test-only, we can probably pack it on the fly. If it's not, we need to make tagged versions of it to supply all demand anyway. So it's looking good.